react-components/src/components/Reveal3DResources/Reveal3DResources.tsx

```diff
@@ -14,20 +25,109 @@ import {
   type TypedReveal3DModel,
   type AddResourceOptions
 } from './types';
+import { type CogniteExternalId } from '@cognite/sdk';
```

Comment:
Can the styling resource moved to separate file? This file is getting bigger with the styling. Will be very useful to maintain a smaller number of line files

Reply:
Will try to extract as helper functions or a hook

Reply:
Will try to extract as helper functions or a hook

Reply:
Created a hook for that